src/utils/index.ts

```diff
@@ -174,7 +174,7 @@ const getClassTypeClassName = (type: string) => {
   }
 }
 
-const getClassType = (type: string) => {
+const getClassType = (type: String) => {
```

Comment:
Why the change from `string` to `String`? According to [this](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types) and [this](https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string), shouldn't it be `string`?